tests/poly_ext/ops.mlir

```diff
@@ -0,0 +1,22 @@
+// RUN: heir-opt %s > %t
+
+// This simply tests for syntax.
+
+#my_poly = #poly.polynomial<1 + x**1024>
+#ring1 = #poly.ring<cmod=7917, ideal=#my_poly>
+#ring2 = #poly.ring<cmod=3123, ideal=#my_poly>
+
+module {
+  func.func @test_ops(%p0 : !poly.poly<#ring1>) {
+    %cmod_switch = poly_ext.cmod_switch %p0 {congruence_modulus=117} : !poly.poly<#ring1> -> !poly.poly<#ring2>
+    return
+  }
+
+  func.func @test_elementwise_ops(%p0 : !poly.poly<#ring1>, %p1: !poly.poly<#ring1>) {
```

Comment:
While I have no problem with keeping this test, in general I think it's not necessary to write tests to exercise purely-upstream capabilities. Unless you did something specific in this PR to enable this beyond adding the ElementwiseMappable trait.

Reply:
I agree that there aren't much "testing" value here. There main reason is a "documentation" to what operation actually looks like, e.g. I frequently refer to `tests/poly/poly_ops.mlir` to copy-paste correct syntax instead of comprehending `assemblyFormat`.